chirp-impl/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,16 @@
+<persistence xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence
+                                 http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd"
+             version="2.1">
+
+    <persistence-unit name="default" transaction-type="RESOURCE_LOCAL">
+        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+        <non-jta-data-source>DefaultDS</non-jta-data-source>
+        <properties>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
+            <property name="hibernate.hbm2ddl.auto" value="update"/>
+        </properties>
```

Comment:
Changed to use `prepareGlobal` and programmatic schema creation.

It might still be tricky in the case of multiple read-sides, because I don't think we make any guarantees about ordering of `prepareGlobal` across multiple event processors, and running it concurrently could be bad.

I think I'll recommend in the documentation that most people will want to create and migrate schemas using an independent process.